content/sigs/docs/gsod/index.adoc

```diff
@@ -0,0 +1,133 @@
+---
+layout: project
+title: "Google Season of Docs in Jenkins"
+section: projects
+sig: docs
+tags:
+- gsod
+- docs
+- docs-sig
+opengraph:
+  image: /images/gsod/gsod.png
+links:
+  meetings: "/projects/gsoc/#office-hours"
+---
+
+image:/images/gsod/gsod.png[Google Season of Docs, role=center, float=right]
+
+The https://developers.google.com/season-of-docs/[Google Season of Docs (GSoD)]
+program brings together open source and technical writers communities for the benefit of both.
+The program raises awareness of open source, of docs, and of technical writing.
+GSoD in Jenkins is organized by the link:/sigs/docs[Documentation special interest group].
+
+== GSoD 2020
+
+Jenkins project will be applying to Google Season of Docs 2020.
+We invite technical writers to join the community and contribute to the documentation being used by millions of Jenkins users worldwide.
+
+=== Project ideas
+
+We invite technical writers to contribute to a major documentation update effort being coordinated in the documentation SIG.
+It includes a number of link:/sigs/docs/#ongoing-projects[ongoing projects],
+and technical writers are welcome to make their proposals in any area of the Jenkins documentation.
+We encourage proposals based on a technical writer's expertise and interests.
+
+[frame="topbot",grid="all",options="header",cols="30%,15%,55%"]
+|=========================================================
+|Project idea | Keywords | Description and links
+
+| Plugin documentation migration and update 
+| Markdown, GitHub, Plugins, Rewrite, Copy-editing
+| Currently we are migrating our plugin documentation from link:https://wiki.jenkins.io/[Confluence] to GitHub,
+  see link:/blog/2019/10/21/plugin-docs-on-github/[this blog post] for announcement.
+  We have already moved hundreds of pages, but there are more than 1000 pages remaining.
+  Migration itself can be done with automated tools we provide, but the documentation usually needs a significant update due to new features.
+  We invite technical writers to select an area (e.g. "plugins for Docker"), and to migrate and update documentation for a number of plugins in such an area.
+  link:/sigs/docs/#plugin-documentation-on-github[More Info]
+
+| Jenkins on Kubernetes
+| AsciiDoc, Guide, Compilation
+| Jenkins on Kubernetes is a popular theme for Jenkins users.
```

Comment:
```suggestion
| Currently Kubernetes is the hottest infrastructure technology on the planet. Jenkins on Kubernetes is a very popular theme for Jenkins users.
```

Reply:
The main aim of this would be to make it more appealing to writers.

Reply:
Fine with me. WDYT @markyjackson-taulia ?

Reply:
I like the change except for the use of `hottest` 😆  and 'on the planet"

Reply:
😆

Reply:
I'd rather not have the change proposed from @sladyn98 .  It feels like stronger marketing language than the Jenkins project generally uses.  I agree that Kubernetes is very interesting to Jenkins users.  I think we should keep the focus on Jenkins and how it can be used on Kubernetes.

Reply:
Yeah no worries sure I was thinking from a different perspective. Apologies if it was not in line with the Jenkins views.